builtin/credential/okta/backend.go

```diff
@@ -267,10 +267,14 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username, pas
 					return nil, logical.ErrorResponse("okta auth backend unexpected failure"), nil, nil
 				}
 
+				timer := time.NewTimer(1 * time.Second)
```

Comment:
In light of my comment above, it might be cleaner to simply do NewTimer, then defer timer.Stop immediately.  Up to you though, we certainly couldn't do that in all cases, just in the cases where we plan to return whenever we Stop.

Reply:
And not even then, if we created the timer within a loop.

Reply:
all cases in this PR are within loops, so didn't use defers